setup.cfg

```diff
@@ -1,7 +1,22 @@
 [flake8]
 max-line-length = 110
 max-doc-length = 79
-ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W504
+# E133: Closing bracket missing indentation
+# E203: whitespace before :, flake8 disagrees with PEP-8
+# E226: missing whitespace around arithmetic operator
+# E228: missing whitespace around bitwise or shift operator
+# N802: function name should be lowercase
+# N803: argument name should be lowercase
+# N806: variable in function should be lowercase
+# N812: lowercase imported as non lowercase
+# N813: camel case imported as lowercase
+# N815: mixed case imported as lowercase
+# N816: mixed case variable in class scope
+# W503: line break after binary operator, flake8 disagrees with PEP-8
+# W504: Line break occurred after a binary operator
+ignore = E133, E203, E226, E228, N802,
+         N803, N806, N812, N813, N815,
+         N816, W503, W504
```

Comment:
I have W503 set in middleware packages and that's the one listed in the developer guide.

Reply:
Unless there's a good reason to remove one of these, I'd probably lean towards keeping it, because it can be very annoying battling these sometimes. Is there a good reason to remove W504, do you think? Or is it more of a case of needing one, but not both?

Reply:
You can't have W503 and W504, you have to pick one for flake8 (they mean opposite things). We pick W504 because that matches black.

Reply:
and then I messed up my comment and said W504 by mistake (now fixed)

Reply:
Well, they're rules to ignore, so I'd have thought you can pick all of them... but sure, will remove W504.